package.json

```diff
@@ -87,5 +87,8 @@
   "bugs": {
     "url": "https://github.com/nhsuk/nhsuk-frontend/issues"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {
+    "puppeteer": "^18.1.0"
```

Comment:
This should be a dev dependency so that it doesn't get installed when people install the package